webServer.js

```diff
@@ -124,8 +132,8 @@ function start() {
     error: getTemplate('templates/500.html', true)
   };
 
-  var generatePage = function (options) {
-    var layout = options.layout || templates.layout;
+  var generatePage = function(options) {
+    var layout = options.layout.content | templates.layout.content;
```

Comment:
Shouldn't this be `||`?
I don't think that bitwise operations on strings are advised..?

Reply:
I'll fix.